python/lsst/ts/criopy/gui/formators.py

```diff
@@ -142,21 +133,77 @@ def reset_formator(self) -> None:
         pass
 
 
+class DataFormator(Formator):
+    """Class formating data prior to displaying them. Internal state of the
+    formator can be reseted by emitting reset_formator signal.
+
+    Parameters
+    ----------
+    field : `str`, optional
+        When specified (and signal parameter is provided), will use this field
+        as fieldname from data arriving with the signal. Defaults to None.
+    fmt : `str`, optional
+        Format string. See Python formatting function for details. Defaults to
+        'd' for decimal number.
+    unit : `astropy.units`, optional
+        Variable unit. Default is None - no unit
+    convert : `astropy.units`, optional
+        Convert values to this unit. Default is None - no unit. If provided,
+        unit must be provided as well.
+    is_warn_func : `func`, optional
+        Function evaluated on each value. If true is returned, value is assumed
+        to be in warning range and will be color coded (displayed in warning
+        text). Default is None - no color coded warning value.
+    is_err_func : `func`, optional
+        Function evaluated on each value. If true is returned, value is assumed
+        to be in error range and will be color coded (displayed in error
+        text). Default is None - no color coded error value.
```

Comment:
The `is_warn_func` and `is_err_func` look like the booleans. How about `warn_func` and `err_func` instead?